### ISSUE TITLE
Build error on install mongoose

### ISSUE THREAD

**Issue by brunobatista** [LearnBoost/mongoose#2561]
I really not know if is a error but when install mongoose this showing me a error:

![bug](https://cloud.githubusercontent.com/assets/441774/5555387/226ffcde-8c93-11e4-9956-8396c0595b16.gif)

```bash
> kerberos@0.0.4 install /home/vagrant/node_modules/mongoose/node_modules/mongodb/node_modules/kerberos          
> (node-gyp rebuild 2> builderror.log) || (exit 0)                                                               
                                                                                                                 
make: Entering directory `/home/vagrant/node_modules/mongoose/node_modules/mongodb/node_modules/kerberos/build'  
  SOLINK_MODULE(target) Release/obj.target/kerberos.node                                                         
make: Leaving directory `/home/vagrant/node_modules/mongoose/node_modules/mongodb/node_modules/kerberos/build'   
-                                                                                                                
> bson@0.2.16 install /home/vagrant/node_modules/mongoose/node_modules/mongodb/node_modules/bson                 
> (node-gyp rebuild 2> builderror.log) || (exit 0)                                                               
                                                                                                                 
make: Entering directory `/home/vagrant/node_modules/mongoose/node_modules/mongodb/node_modules/bson/build'      
  CXX(target) Release/obj.target/bson/ext/bson.o                                                                 
make: Leaving directory `/home/vagrant/node_modules/mongoose/node_modules/mongodb/node_modules/bson/build'       
```

Thanks to try help me.

**Comment by vkarpov15** [comment]
This isn't an error as far as I can tell, this is rebuilding the [kerberos](https://www.npmjs.com/package/kerberos) and [bson](https://www.npmjs.com/package/bson) modules, which use native C++ extensions.

**Comment by brunobatista** [comment]
Ok, thanks @vkarpov15 :+1:

**Comment by billwheaton1715** [comment]
I've been working on trying to get this to work all week on my windows 7 machine.
You have to have Python installed.  It creates the VC project files that are needed for the node-gyp build.  Then you have to have moft VC10 installed with the Windows 7 C++ SDK.
see this blog post about that.  http://blogs.msdn.com/b/vcblog/archive/2011/03/31/10148110.aspx.  To get it to work, I ended up uninstalling anything to do with moft development and starting over after manually deleting directories that uninstall missed, and all the tmp files, and downloading CCleaner to clean my registry and creating a new restore point (have I mentioned how much I hate moft?)  Then finally about two hours ago I found out that Kerebos has a new version 0.0.9 that compiles in windoze while mongodb still uses 0.0.7 (which does not compile)

So.  second to bottom line.... bson seems to be fine.  it has compile warning, and the testing fails because it can't find Kernal32.dll.  But its there.  and even if it isn't, there is a "pure javascript" version as a backup plan.

Bottom line - mongodb needs to update the package.json file to reference kerebos@0.0.9 instead of 0.0.7 in order for it to compile...  It has warnings too (because of xlocale try/catch), but its a whole lot better than the old one.

I'm new to this whole git thing and fixing stuff, so I may cut my teeth with this
